prompts/generators/randomprompt.py

```diff
@@ -6,9 +6,34 @@
 from prompts import constants
 from prompts.wildcardmanager import WildcardManager
 from . import PromptGenerator, re_combinations, re_wildcard
+from modules.shared import opts,state
 
-logger = logging.getLogger(__name__)
+is_debug = getattr(opts, "is_debug", False)
```

Comment:
Is is_debug a standard part of auto1111 or did you add it? I can't find reference to it in the code

Reply:
![2022-12-01 20 51 25](https://user-images.githubusercontent.com/20321215/205046268-ecf7711b-ef94-4b1f-aabb-8e49098aab6f.png)

I use it like this.

The part processed so that it does not affect others.

Treat it according to your taste.

Got a hint here.
https://github.com/adieyal/sd-dynamic-prompts/blob/98528f4acb4615d830392de5edc448e991742f4c/scripts/dynamic_prompting.py#L39

Reply:
great - it's a good idea